packages/cli/config/eslint.backend.js

```diff
@@ -54,11 +54,10 @@ module.exports = {
     'no-console': 0, // Permitted in console programs
     'new-cap': ['error', { capIsNew: false }], // Because Express constructs things e.g. like 'const r = express.Router()'
     'import/newline-after-import': 'error',
-    'import/no-duplicates': 'warn',
     'import/no-extraneous-dependencies': [
       'error',
       {
-        devDependencies: false,
+        devDependencies: ['**/*.test.*', 'src/setupTests.*', 'dev/**'],
```

Comment:
@Rugvip it should be **/src/setupTests.*  to lookup recursively or am I missing something here? I'm getting some errors in setupTests for our packages/plugins and I can not override this via eslintrc.js (root: true) in the root path.

Reply:
This really should be relative to the package directory, but I'm also finding that that's not always the case. The downside of making it recursive is that especially `dev/` might appear in other places where these rules shouldn't be applied.

I think it's the way we need to go though, would you like to contribute a fix?

Reply:
Sure, I will create a PR today